README.md

```diff
@@ -11,3 +11,233 @@ TODO: description
 ## API Routes
 
 TODO: document API routes
+
+#### `GET /`
+
+This renders the `index.html` file that will be used to interact with the backend
+
+### Insurance Card 
+
+### `GET /api/insuranceCards?owner=USERNAME` - Get insurance cards by owner
+
+**Returns**
+
+- An array of insurance cards created by user with username `owner` in alphabetical order with respect to their purpose
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
```

Comment:
Should there be another check for if the owner is the currently logged in user? Logic being that users should only be able to access their own info?